### DISCUSSION TITLE
Installing as root with no YouTube app

### DISCUSSION THREAD

**Discussion by Turhvjbufv** [anddea/revanced-patches#439] [Q&A] [answered]
What patches should I exclude?
And let's say I don't even have YouTube on my phone (custom rom) should I still mount or just normal install?

**Comment by anddea** [comment, marked as answer]
Exclude microG support and custom package name. I believe you have to install with mount for root.

**Comment by Turhvjbufv** [comment]
That worked, but also just for anyone coming here, using micro g you need to disable authenticae with device and enable strip device name(both in micro g settings) or just use play service 
And you need to install the normal YouTube app before so that rvx can mount it

**Comment by Turhvjbufv** [comment]
Oh and idk why but root revanced extended is slower than non root

**Reply by anddea** [reply]
I think it's a coincidence, there should be no difference, maybe just give a time for system to adapt, reboot the device to speed it up (note: if mount was not successful then the YT app will revert to original again after reboot). The same thing goes when you install different mods and different patches. Also, when a lot of additional features are enabled (except for some forcible patches), they may slow down the app, that's why I recommend to use only important features for you. 

That's also the reason I created this thread, so people can provide ideas to change defaults:
https://github.com/anddea/revanced-patches/discussions/308
